### PR TITLE
CI for LTS

### PR DESCRIPTION
**Pull request by ChrisRackauckas**

### CODE REVIEW

**Comment by ChrisRackauckas** [comment]
@KirillZubov can you take it?

**Review by KirillZubov — DISMISSED** [review]

**Review by KirillZubov — APPROVED** [review]

**Comment by KirillZubov** [comment]
@ChrisRackauckas it's ready for merge, I guess

**Comment by ChrisRackauckas** [comment]
Great, thanks!

File: test/neural_adapter_tests.jl [hunk 1]
```diff
@@ -195,7 +195,8 @@ for i in 1:count_decomp
     phi_in(cord) = phis[i-1](cord,reses[i-1].minimizer)
     # phi_bound(x,y) = if (x isa Matrix)  phi_in(vcat(x, fill(y,size(x)))) else  phi_in(vcat(fill(x,size(y)),y)) end
     phi_bound(x,y) = phi_in(vcat(x,y))
-    @register phi_bound(x,y)
+    @register_symbolic phi_bound(x,y)
```
Comment (killah-t-cell) [inline test/neural_adapter_tests.jl:198]: is this a new way to register functions with modelingToolkit?

Comment (ChrisRackauckas) [reply test/neural_adapter_tests.jl:198]: yeah there's a deprecation

**Pull request merged**